rails/app/controllers/api/v1/report_learners_es_controller.rb

```diff
@@ -271,4 +338,26 @@ def self.query_es(options, user)
       :headers => { 'Content-Type' => 'application/json' } )
     return esSearchResult
   end
+
+  def self.detailed_learner_info(learner)
+    {
+      student_id: learner.student_id,
+      learner_id: learner.learner_id,
+      class_id: learner.class_id,
+      class: learner.class_name,
+      school: learner.school_name,
+      user_id: learner.user_id,
+      permission_forms: learner.permission_forms,
+      username: learner.username,
+      student_name: learner.student_name,
+      teacher_name: learner.teachers_name,
+      teacher_id: learner.teachers_id,
+      teacher_district: learner.teachers_district,
+      teacher_state: learner.teachers_state,
+      teacher_email: learner.teachers_email,
```

Comment:
The queries that Matt created expect a teachers property with an array of object one per teacher and then each object has the name, id, district, state, and email. 

Fixing this doesn't have to be in this PR. And maybe it would be easier to just do that translation in the query-creator.

Reply:
Fixed in the portal, for one teacher. Kicking the can down the road for how we will find the other teachers.